### PR TITLE
Always keep the last etenergies and similar attributes

### PR DESCRIPTION
**Pull request by langner**

### CODE REVIEW

**Comment by langner** [comment]
This is for #740.

**Comment by langner** [comment]
Oops, seems this breaks a bunch of tests.

**Comment by langner** [comment]
Actually, this breaks 4-6 new tests, there were already a few others broken. See #742 for that.

File: cclib/parser/orcaparser.py [hunk 1]
```diff
@@ -796,9 +795,13 @@ def splitter(line):
                         contrib = numpy.nan
                     sec.append([start, end, contrib])
                     line = next(inputfile)
-                self.etsecs.append(sec)
+                etsecs.append(sec)
                 line = next(inputfile)
 
+            self.etenergies = etenergies
+            self.etsecs = etsecs
+            self.etsyms = etsyms
```
Comment (berquist) [inline cclib/parser/orcaparser.py]: Maybe use `set_attribute` for these three?

Comment (langner) [reply cclib/parser/orcaparser.py]: Done, and in several other places.

**Comment by berquist** [comment]
Poke for rebase, but there's also another issue:
```
    from cclib.parser.orcaparser import ORCA
E     File "/home/travis/build/cclib/cclib/cclib/parser/orcaparser.py", line 1109
E       if line[:23] == "MULLIKEN ATOMIC CHARGES":
E                                                ^
E   SyntaxError: invalid syntax
```

**Comment by langner** [comment]
> Poke for rebase, but there's also another issue:
> 
> ```
>     from cclib.parser.orcaparser import ORCA
> E     File "/home/travis/build/cclib/cclib/cclib/parser/orcaparser.py", line 1109
> E       if line[:23] == "MULLIKEN ATOMIC CHARGES":
> E                                                ^
> E   SyntaxError: invalid syntax
> ```

Should be good now.

**Comment by langner** [comment]
> > Poke for rebase, but there's also another issue:
> > ```
> >     from cclib.parser.orcaparser import ORCA
> > E     File "/home/travis/build/cclib/cclib/cclib/parser/orcaparser.py", line 1109
> > E       if line[:23] == "MULLIKEN ATOMIC CHARGES":
> > E                                                ^
> > E   SyntaxError: invalid syntax
> > ```
> 
> Should be good now.

Actually wasn't; I broke three attributes with my refactoring. That should be fixed now, and added the method `extend_attribute` onto parsers in the spirit of existing methods.

**Comment by langner** [comment]
> Right now, keeping the last of everything is fine, but in the future we should keep all instances of all properties. However, only keeping the last is better than only keeping the first.

If you have a specific action item related to that, please go ahead and create a subsequent issue :)

**Pull request merged**